clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/config/Justification.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Lumberjack authors (see AUTHORS file)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.abcxyz.lumberjack.auditlogclient.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class Justification {
+  private static final String JUSTIFICATION_PUBLIC_KEYS_ENDPOINT_ENV_KEY =
+      "AUDIT_CLIENT_JUSTIFICATION_PUBLIC_KEYS_ENDPOINT";
+  private static final String JUSTIFICATION_ENABLED_ENV_KEY = "AUDIT_CLIENT_JUSTIFICATION_ENABLED";
+
+  @JsonProperty("public_keys_endpoint")
+  private String publicKeysEndpoint = "localhost:8080";
```

Comment:
To be consistent with the logic in Golang part(https://github.com/abcxyz/lumberjack/pull/284 fyi), we no longer use "localhost:8080" as the default value. There is no default value for the public_keys_endpoint

Reply:
Any particular reason that we should not use this default value?

Reply:
"localhost:8080" doesn't make sense. In production, people won't use "localhost:8080" to connect to jvs

Reply:
+1 We no longer set default for this value in Go https://github.com/abcxyz/lumberjack/blob/main/clients/go/apis/v1alpha1/config.go#L358

Reply:
Ack, thanks!